ansible/roles/docker/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 # role variable for the swarm service
 
 docker_api_port: 2385
+thin_provisioned: false 
```

Comment:
it's not a default though, it's a var. it doesn't leave the role and we 
don't want it set outside of the role.

On 26 Feb 2016, at 10:23, Madhav Puri wrote:

>> @@ -2,3 +2,4 @@
>> # role variable for the swarm service
>>
>> docker_api_port: 2385
>> +thin_provisioned: false
>
> this file has been removed in `ansible/master` as part of fixing 
> https://github.com/contiv/ansible/issues/61, we will need to move the 
> new variable in `roles/docker/defaults/main.yml` file
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/152/files#r54280846